src/redux/reducers/index.js

```diff
@@ -1,3 +1,5 @@
+import 'react-native-get-random-values';
+import { v4 as uuidv4 } from 'uuid';
 import { actionTypes } from '../epics';
 import processResource from './process-resources';
 import RESOURCE_TYPES from '../../resources/resourceTypes';
```

Comment:
looks like two new constants were added to origin/master:
PLURAL_RESOURCE_TYPES and SINGULAR_RESOURCE_TYPES